test/swe2d/test_atmospheric_pressure.py

```diff
@@ -0,0 +1,72 @@
+# Atmospheric pressure test case
+# Constant pressure gradient applied at surface
+# Result compared to analytic solution

```

Comment:
The description could be better: you force the flow in a rectangular channel with a constant atm pressure gradient until the solution reaches a steady state? In the steady state the elevation gradient balances out the pressure gradient, and you compare the elevation to the analytical solution?